### ISSUE TITLE
False Positive - Missing top-level documentation comment

### ISSUE THREAD

**Issue by MartinSGill** [bbatsov/rubocop#1949]
When a ruby file contains trailing comments after the `end` of a block rubocop incorrectly reports the class has missing top-level documentation comments.

example.rb:

```ruby
# My Module
module MyModule
  # My Class
  class MyClass
    def initialize
    end
  end # end of my class
end # end of my module
```

rubcop output:

```text
> rubocop --version
0.31.0

> rubocop example.rb
Inspecting 1 file
C

Offenses:

example.rb:4:3: C: Missing top-level class documentation comment.
  class MyClass
  ^^^^^
```

remove the trailing comments from the `end` block and rubocop now correctly recognises the existing top-level comments.

example2.rb:
```ruby
# My Module
module MyModule
  # My Class
  class MyClass
    def initialize
    end
  end
end
```

rubocop output:

```text
> rubocop example2.rb
Inspecting 1 file
.

1 file inspected, no offenses detected
```

**Comment by bbatsov** [comment]
I think we fixed this on `master`. See if you can reproduce the problem there.
